manifests/agent.pp

```diff
@@ -31,18 +31,17 @@
 #     method        => 'service',
 #  }
 #
-class puppet::agent(
+class puppet::agent (
   $ensure            = 'present',
   $server            = 'puppet',
   $ca_server         = undef,
   $report            = true,
   $report_server     = undef,
-  $report_format     = undef,
   $manage_repos      = true,
   $environment       = $::environment,
   $pluginsync        = true,
   $certname          = $::clientcert,
-  $showdiff          = true,
+  $showdiff          = false,
```

Comment:
How about renaming this to show_diff to keep it consistent with the actual config variable?